android/settings.gradle

```diff
@@ -1,5 +1,8 @@
 rootProject.name = 'bitcampapp'
-
-include ':app'
 include ':react-native-vector-icons'
 project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
+include ':react-native-svg'
+project(':react-native-svg').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-svg/android')
+include ':app'
+include ':PDFView'
+project(':PDFView').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-pdf-view/android')
```

Comment:
Are we using these libraries (react-native-svg, pdfview)? If not, we can take out a lot of this stuff.